contributors/design-proposals/container-runtime-interface-v1.md

```diff
@@ -154,7 +154,7 @@ react to changes in lifecycle.
 The only transitions that are possible for a container are described below:
 
 ```
-() -> Created        // A container can only transition to created from the
+() -> Created        // A container can only transition to create from the
```

Comment:
I agree with you, but now how should I do?  Shoud I correct and pull request again?